### PR TITLE
Remove hidden whitespace before submitting

### PR DESCRIPTION
**Pull request by diversemix**
#### Background

Removes whitespace from user entered fields

#### Any relevant tickets

Partially for #2401

#### How has this been tested?
- [x] Unit / Integration tests

#### UI Changes

N/A

### CODE REVIEW

**Review by erezmus — APPROVED** [review]

File: packages/component-model-manuscript/entities/manuscript/index.js [hunk 1]
```diff
@@ -315,6 +315,29 @@ class Manuscript extends BaseModel {
...
+
+  static cleanWhitespace(input) {
+    const teamsToClean = ['opposedReviewers', 'suggestedReviewers']
+
+    /* eslint no-param-reassign: ["error", { "props": false }] */
+    teamsToClean.forEach(team => {
+      if (input[team]) {
+        input[team].forEach(item => {
+          if (item.email) {
+            item.email = item.email.trim()
```
Comment (Cooryd) [inline packages/component-model-manuscript/entities/manuscript/index.js:334]: 👍

**Review by Cooryd — APPROVED** [review]

File: packages/component-model-manuscript/entities/manuscript/index.js [hunk 2]
```diff
@@ -324,6 +324,7 @@ class Manuscript extends BaseModel {
     const teamsToClean = ['opposedReviewers', 'suggestedReviewers']
 
     /* eslint no-param-reassign: ["error", { "props": false }] */
+    input.author.email = input.author.email.trim()
```
Comment (Cooryd) [inline packages/component-model-manuscript/entities/manuscript/index.js]: Are we happy there will always be an email property on author to call `trim` on?

Comment (diversemix) [reply packages/component-model-manuscript/entities/manuscript/index.js]: :thinking: fair point - I'll change

**Review by Cooryd — APPROVED** [review]
LGTM 👍

**Review by norisalsaadie — APPROVED** [review]

**Pull request merged**